src/components/controls/TransitToggleButton.tsx

```diff
@@ -30,15 +31,16 @@ class TransitToggleButton extends React.Component
 
     public render() {
         return (
-            <button
+            <div
                 className={classNames(
                     s.button,
                     this.getToggledButtonClass(this.props.type, this.props.toggled),
+                    !this.props.disabled ? s.active : undefined,
```

Comment:
A bad practice is to have active as a "special case". Here active means "normal state"

Instead, add cursor: pointer to .button and use code:

```
this.props.disabled ? s.disabled : undefined

.disabled {
     pointer-events: none;
     cursor: none;
}
```